.github/workflows/build_and_test.yml

```
@@ -164,8 +183,8 @@ jobs:
         commit_id: ${{ steps.determine_commit_id_and_email_author.outputs.COMMIT_ID }}
```

actionlint (1.7.10)

property "determine_commit_id_and_email_author" is not defined in object type {check_commit_message: {conclusion: string; outcome: string; outputs: {string => string}}; check_existing_image: {conclusion: string; outcome: string; outputs: {string => string}}; get_job_id: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)